stackrox/stackrox
# chore(ui): Delete ROX_NVD_CVSS_UI feature flag in ui

### Description

This (frontend only) feature flag has been turned on since 2024-10-28 in #13063

Version 4.6.0 was released on 2024-12-03

Therefore, delete feature flag before changes for **Advisories** or **EPSS** columns in some of the same files.

https://github.com/stackrox/stackrox/tree/master/ui/apps/platform#delete-a-feature-flag-from-frontend-code

### User-facing documentation


### Testing and quality


#### Automated testing


#### How I validated my change

1. `npm run lint` in ui/apps/platform
2. `npm run build` in ui/apps/platform and then compute branch - master for the following
    * `wc build/static/js/*.js`
        main.js 0 = 4615653 - 4615653
        total -218 = 11561621 - 11561839
    * `ls -al build/static/js/*.js | wc`
        files 0 = 179 - 179
3. `npm run start` in ui/apps/platform

#### Manual testing

Local deployment without `ROX_SCANNER_V4` feature flag.

1. Visit /main/vulnerabilities/workload-cves
    WorkloadCVEOverviewTable
    See absence in local without and presence on staging with: **Top NVD CVSS** column

2. Click a vulnerability
    AffectedImagesTable
    See absence in local without and presence on staging with: **NVD CVSS** column

3. Click an image
    ImageVulnerabilitiesTable
    See absence in local without and presence on staging with: **NVD CVSS** column

4. Visit /main/vulnerabilities/reports and then create a report configuration
    ReportParametersForm
    See absence in local without and presence on staging with: **Include NVD CVSS** checkbox

5. Visit /main/policy-management/policies create a policy, advance to **Rules**, and then expand **Image contents**
    policyCriteriaDescriptors
    See presence in both local without and on staging with: **NVD CVSS** field

6. Drag **NVD CVSS** field to rule.
    See presence in both local without and on staging with: **NVD CVSS scores require Scanner V4** info text
